roachtest: django failed

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2177056&tab=buildLog) on [master@457f1af5d19e0bcef9b5ea9e9125b3332717a86c](https://github.com/cockroachdb/cockroach/commits/457f1af5d19e0bcef9b5ea9e9125b3332717a86c):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:218,orm_helpers.go:144,django.go:204,django.go:216,test_runner.go:754: 
		Tests run on Cockroach v20.2.0-alpha.1-2102-g457f1af5d1
		Tests run against django cockroach-3.0.x
		8032 Total Tests Run
		8031 tests passed
		1 test failed
		489 tests skipped
		20 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: get_or_create.tests.UpdateOrCreateTransactionTests.test_creation_in_transaction (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_2) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2177056&tab=artifacts#/django)
Related:
- #52761 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2180641&tab=buildLog) on [master@1049eaa26d983a0449049264d6c2ee2cc27454e2](https://github.com/cockroachdb/cockroach/commits/1049eaa26d983a0449049264d6c2ee2cc27454e2):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:218,orm_helpers.go:144,django.go:204,django.go:216,test_runner.go:754: 
		Tests run on Cockroach v20.2.0-alpha.1-2176-g1049eaa26d
		Tests run against django cockroach-3.0.x
		8032 Total Tests Run
		8030 tests passed
		2 tests failed
		489 tests skipped
		20 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: get_or_create.tests.UpdateOrCreateTransactionTests.test_creation_in_transaction (unexpected)
		--- FAIL: expressions.tests.BasicExpressionsTests.test_case_in_filter_if_boolean_output_field (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_2) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2180641&tab=artifacts#/django)
Related:
- #52761 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2184311&tab=buildLog) on [master@d106b79cb3838bbc1fb163e8d8216488130b521e](https://github.com/cockroachdb/cockroach/commits/d106b79cb3838bbc1fb163e8d8216488130b521e):

```
	orm_helpers.go:218,orm_helpers.go:144,django.go:204,django.go:216,test_runner.go:754: 
		Tests run on Cockroach v20.2.0-alpha.1-2216-gd106b79cb3
		Tests run against django cockroach-3.0.x
		6547 Total Tests Run
		6538 tests passed
		9 tests failed
		412 tests skipped
		12 tests ignored
		0 tests passed unexpectedly
		9 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: queries.test_bulk_update.BulkUpdateTests.test_only_concrete_fields_allowed (unexpected)
		--- FAIL: queries.tests.UnionTests.test_AB_ACB (unexpected)
		--- FAIL: queries.tests.UnionTests.test_BAB_BACB (unexpected)
		--- FAIL: queries.tests.UnionTests.test_BAB_BAC (unexpected)
		--- FAIL: queries.tests.UnionTests.test_A_AB2 (unexpected)
		--- FAIL: queries.tests.UnionTests.test_A_AB (unexpected)
		--- FAIL: queries.tests.UnionTests.test_BA_BCA__BAB_BAC_BCA (unexpected)
		--- FAIL: get_or_create.tests.UpdateOrCreateTransactionTests.test_creation_in_transaction (unexpected)
		--- FAIL: queries.test_bulk_update.BulkUpdateTests.test_large_batch (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_2) is available in the artifacts' django log

	cluster.go:1612,context.go:135,cluster.go:1601,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2184311-1597472915-38-n1cpu16 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1808
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1357
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2184311&tab=artifacts#/django)
Related:
- #52761 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2195516&tab=buildLog) on [master@ebd5c732f83009acc9c6f5859ca95e74e5453a1c](https://github.com/cockroachdb/cockroach/commits/ebd5c732f83009acc9c6f5859ca95e74e5453a1c):

```
	orm_helpers.go:218,orm_helpers.go:144,django.go:204,django.go:216,test_runner.go:754: 
		Tests run on Cockroach v20.2.0-alpha.1-2373-gebd5c732f8
		Tests run against django cockroach-3.0.x
		6548 Total Tests Run
		6539 tests passed
		9 tests failed
		411 tests skipped
		12 tests ignored
		0 tests passed unexpectedly
		9 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: queries.test_bulk_update.BulkUpdateTests.test_only_concrete_fields_allowed (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: queries.tests.UnionTests.test_BAB_BACB (unexpected)
		--- FAIL: queries.tests.UnionTests.test_BAB_BAC (unexpected)
		--- FAIL: queries.tests.UnionTests.test_BA_BCA__BAB_BAC_BCA (unexpected)
		--- FAIL: get_or_create.tests.UpdateOrCreateTransactionTests.test_creation_in_transaction (unexpected)
		--- FAIL: queries.tests.UnionTests.test_AB_ACB (unexpected)
		--- FAIL: queries.tests.UnionTests.test_A_AB (unexpected)
		--- FAIL: queries.tests.UnionTests.test_A_AB2 (unexpected)
		--- FAIL: queries.test_bulk_update.BulkUpdateTests.test_large_batch (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_2) is available in the artifacts' django log

	cluster.go:1612,context.go:135,cluster.go:1601,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2195516-1597818221-37-n1cpu16 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1808
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1357
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2195516&tab=artifacts#/django)
Related:
- #52973 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

- #52879 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202008151325_v19.2.10](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202008151325_v19.2.10)

- #52761 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2200586&tab=buildLog) on [master@9f06285518c03e00d925725aaeaa2d5687b4e2ac](https://github.com/cockroachdb/cockroach/commits/9f06285518c03e00d925725aaeaa2d5687b4e2ac):

```
	orm_helpers.go:218,orm_helpers.go:144,django.go:204,django.go:216,test_runner.go:754: 
		Tests run on Cockroach v20.2.0-alpha.1-2453-g9f06285518
		Tests run against django cockroach-3.0.x
		6548 Total Tests Run
		6539 tests passed
		9 tests failed
		411 tests skipped
		12 tests ignored
		0 tests passed unexpectedly
		9 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: queries.test_bulk_update.BulkUpdateTests.test_large_batch (unexpected)
		--- FAIL: queries.tests.UnionTests.test_BAB_BAC (unexpected)
		--- FAIL: get_or_create.tests.UpdateOrCreateTransactionTests.test_creation_in_transaction (unexpected)
		--- FAIL: queries.tests.UnionTests.test_BAB_BACB (unexpected)
		--- FAIL: queries.tests.UnionTests.test_AB_ACB (unexpected)
		--- FAIL: queries.tests.UnionTests.test_A_AB2 (unexpected)
		--- FAIL: queries.tests.UnionTests.test_BA_BCA__BAB_BAC_BCA (unexpected)
		--- FAIL: queries.test_bulk_update.BulkUpdateTests.test_only_concrete_fields_allowed (unexpected)
		--- FAIL: queries.tests.UnionTests.test_A_AB (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_2) is available in the artifacts' django log

	cluster.go:1612,context.go:135,cluster.go:1601,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2200586-1597904987-36-n1cpu16 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1808
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1357
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2200586&tab=artifacts#/django)
Related:
- #53098 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202008191705_v19.2.10](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202008191705_v19.2.10) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #52973 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2)

- #52761 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2219091&tab=buildLog) on [master@300dbe9a958a9461f320e262b0929cf2c44b7c24](https://github.com/cockroachdb/cockroach/commits/300dbe9a958a9461f320e262b0929cf2c44b7c24):

```
		Tests run on Cockroach v20.2.0-alpha.1-2711-g300dbe9a95
		Tests run against django cockroach-3.0.x
		6547 Total Tests Run
		6537 tests passed
		10 tests failed
		411 tests skipped
		13 tests ignored
		0 tests passed unexpectedly
		10 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: queries.tests.UnionTests.test_AB_ACB (unexpected)
		--- FAIL: get_or_create.tests.UpdateOrCreateTransactionTests.test_creation_in_transaction (unexpected)
		--- FAIL: queries.tests.UnionTests.test_BAB_BACB (unexpected)
		--- FAIL: queries.test_bulk_update.BulkUpdateTests.test_large_batch (unexpected)
		--- FAIL: queries.tests.UnionTests.test_BAB_BAC (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: queries.tests.UnionTests.test_A_AB2 (unexpected)
		--- FAIL: expressions.tests.BasicExpressionsTests.test_case_in_filter_if_boolean_output_field (unexpected)
		--- FAIL: queries.test_bulk_update.BulkUpdateTests.test_only_concrete_fields_allowed (unexpected)
		--- FAIL: queries.tests.UnionTests.test_A_AB (unexpected)
		--- FAIL: queries.tests.UnionTests.test_BA_BCA__BAB_BAC_BCA (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_2) is available in the artifacts' django log

	cluster.go:1612,context.go:135,cluster.go:1601,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2219091-1598335789-36-n1cpu16 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | main.glob..func14
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1143
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:267
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1839
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1357
		Wraps: (3) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *errutil.leafError

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2219091&tab=artifacts#/django)
Related:
- #53381 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202008241848_v20.1.5](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202008241848_v20.1.5) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #52761 roachtest: django failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/sql/sem/tree/constant.go
+++ b/pkg/sql/sem/tree/constant.go
@@ -368,13 +368,18 @@ func (expr *NumVal) ResolveAsType(
 	}
 }
 
+// intersectTypeSlices returns a slice of all the types that are in both of the
+// input slices that have the same OID.
 func intersectTypeSlices(xs, ys []*types.T) (out []*types.T) {
+	seen := make(map[oid.Oid]struct{})
 	for _, x := range xs {
 		for _, y := range ys {
-			if x == y {
+			_, ok := seen[x.Oid()]
+			if x.Oid() == y.Oid() && !ok {
 				out = append(out, x)
 			}
 		}
+		seen[x.Oid()] = struct{}{}
 	}
 	return out
 }
@@ -458,7 +463,6 @@ var (
 		types.String,
 		types.Bytes,
 		types.Bool,
-		types.Box2D,
 		types.Int,
 		types.Float,
 		types.Decimal,

--- a/pkg/sql/sem/tree/constant_test.go
+++ b/pkg/sql/sem/tree/constant_test.go
@@ -26,7 +26,34 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/types"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
+	"github.com/lib/pq/oid"
 )
+
+// TestAvailTypesAreSets verifies that all of the constant "available type"
+// slices don't have duplicate OIDs.
+func TestAvailTypesAreSets(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+	testCases := []struct {
+		availTypes []*types.T
+	}{
+		{tree.NumValAvailInteger},
+		{tree.NumValAvailDecimalNoFraction},
+		{tree.NumValAvailDecimalWithFraction},
+		{tree.StrValAvailAllParsable},
+		{tree.StrValAvailBytes},
+	}
+
+	for i, test := range testCases {
+		seen := make(map[oid.Oid]struct{})
+		for _, newType := range test.availTypes {
+			if _, ok := seen[newType.Oid()]; ok {
+				t.Errorf("%d: found duplicate type: %v", i, newType)
+			}
+			seen[newType.Oid()] = struct{}{}
+		}
+	}
+}
 
 // TestNumericConstantVerifyAndResolveAvailableTypes verifies that test NumVals will
 // all return expected available type sets, and that attempting to resolve the NumVals

